src/Test/test_constraint.jl

```diff
@@ -16,6 +16,7 @@ function test_constraint_get_ConstraintIndex(
     model::MOI.ModelLike,
     ::Config{T},
 ) where {T}
+    @requires MOI.supports(model, MOI.ConstraintName(), MOI.ConstraintIndex)
```

Comment:
I don't think this is quite right, because every solver should fail the `MOI.ConstraintIndex` check. 

It's asking for a method like `MOI.supports(::MOI.ModelLike, ::MOI.ConstraintName, ::Type{MOI.ConstraintIndex})` which is different to `MOI.supports(::MOI.ModelLike, ::MOI.ConstraintName, ::Type{MOI.ConstraintIndex{F,S}) where {F,S}`.

But we could use explicit `F = MOI.ScalarAffineFunction{T}`  and `S = MOI.GreaterThan{T}` (and another for `S=MOI.LessThan{T}`.

Reply:
just did